homeassistant/components/iqvia/sensor.py

```diff
@@ -219,5 +224,9 @@ class IndexSensor(IQVIAEntity):
                     '{0}_{1}'.format(ATTR_ALLERGEN_AMOUNT, index):
                         attrs['PPM'],
                 })
+        elif self._type in (TYPE_DISEASE_TODAY, TYPE_DISEASE_YESTERDAY):
+            for attrs in period['Triggers']:
+                self._attrs['{0}_index'.format(
+                    attrs['Name'].lower())] = attrs['Index']
```

Comment:
Isn't this adding historical data to the state machine?

Reply:
I'm thinking this is "history":
`TYPE_DISEASE_YESTERDAY`

Reply:
Oop, mistype on my part. Thanks!